Doc/library/os.rst

```diff
@@ -741,6 +741,9 @@ as internal buffering of data.
    <fd_inheritance>` by default or non-inheritable if *inheritable*
    is ``False``.
 
+   If *fd* is valid and equal to *fd2*, this function has no effect
```

Comment:
There should be a `versionchanged` directive.

Reply:
"this function has no effect": that's wrong, it does validate the FD which is a legit usage of this function. I would rather remove this sentence.